### PR TITLE
engine events: fix event names in rules or enum - v2

### PR DESCRIPTION
**Pull request by jasonish**
Changes:
- Update rule rev.

Fixes these warnings when using engine events:
```
Warning: detect-app-layer-event: app-layer-event keyword's protocol "dns" doesn't have event "not_a_request" registered [DetectAppLayerEventSetup:detect-app-layer-event.c:262]
Warning: detect-app-layer-event: app-layer-event keyword's protocol "dns" doesn't have event "not_a_response" registered [DetectAppLayerEventSetup:detect-app-layer-event.c:262]
Warning: detect-app-layer-event: app-layer-event keyword's protocol "http2" doesn't have event "invalid_http1_settings" registered [DetectAppLayerEventSetup:detect-app-layer-event.c:262]
Warning: detect-app-layer-event: app-layer-event keyword's protocol "ike" doesn't have event "weak_crypto_nodh" registered [DetectAppLayerEventSetup:detect-app-layer-event.c:262]
Warning: detect-app-layer-event: app-layer-event keyword's protocol "ike" doesn't have event "weak_crypto_noauth" registered [DetectAppLayerEventSetup:detect-app-layer-event.c:262]
Warning: detect-app-layer-event: app-layer-event keyword's protocol "modbus" doesn't have event "invalid_unit_identifier" registered [DetectAppLayerEventSetup:detect-app-layer-event.c:262]
```

Ticket: https://redmine.openinfosecfoundation.org/issues/7361

### CODE REVIEW

**Comment by suricata-qa** [comment]
Information: QA ran without warnings. 

Pipeline 23239

**Comment by victorjulien** [comment]
I think this looks good as a fix, but I was wondering if we should fix the rules or the code? We can make the existing rules work by changing the event names in the code, right? Or would we risk breaking ppls working rules that way?

**Review by victorjulien — APPROVED** [review]

**Comment by victorjulien** [comment]
Merged in #12088, thanks!

File: rust/src/http2/http2.rs [hunk 1]
```diff
@@ -498,7 +498,7 @@ pub enum HTTP2Event {
     ExtraHeaderData,
     LongFrameData,
     StreamIdReuse,
-    InvalidHTTP1Settings,
+    InvalidHttp1Settings,
```
Comment (catenacyber) [inline rust/src/http2/http2.rs:501]: Thanks for this one Jason :-)

**Pull request closed without merging**